src/core/config/config.go

```diff
@@ -478,3 +479,15 @@ func GetPermittedRegistryTypesForProxyCache() []string {
 	}
 	return strings.Split(types, ",")
 }
+
+// GetGCBlobTimeWindow returns the reserve time window of blob.
+func GetGCBlobTimeWindow() int64 {
+	// the env is for testing/debugging. For production, Do NOT set it.
+	if env, exist := os.LookupEnv("_GC_BLOB_TIME_WINDOW"); exist {
```

Comment:
`_GC_BLOB_TIME_WINDOW_HOURS`. Do we need `BLOB` in this env ? or `_GC_TIME_WINDOW_HOURS`?

Reply:
+1
and I don't think we need the first underscore

Reply:
updated